<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>560</width>
     <height>510</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>280</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>280</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 80);
border-top-left-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>30</y>
      <width>251</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 255);
border-bottom-right-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>70</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0, 0, 0, 200);</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_id">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>  User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_pwd">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>  Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_login">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>299</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}

QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}</string>
    </property>
    <property name="text">
     <string>L o g  I n</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>437</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255,229,0,255)</string>
    </property>
    <property name="text">
     <string>hanbisw@naver.com</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>81</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 75);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>368</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0, 0, 0, 210);</string>
    </property>
    <property name="text">
     <string>Forgot your User Name or password?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_close">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>33</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>x.png</normaloff>x.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
